manager/resource_test.go

```diff
@@ -1,21 +1,23 @@
-package manager
+package manager_test
```

Comment:
why do you change the package?

Reply:
To avoid cyclic imports, as the new `test/util` subpackage now imports from manager.